stubs/extensions/reflection-generated/dom-8.1.2.phpstub

```diff
@@ -0,0 +1,1019 @@
+<?php
+// Generated with PHP version 8.1.2 dom version 20031129
```

Comment:
I named the file dom-8.1.2 because I'm pretty sure this dom version is misleading and the dom extension has actually changed along with php. I'm not sure why it doesn't show the php version like the other php-bundled extensions, but I'm guessing that's a bug on their end.